app/controllers/visns_controller.rb

```diff
@@ -73,6 +73,10 @@ def show
 
   def load_facilities_show_rows
     @facilities = VaFacility.get_by_visn(@visn).select(:common_name, :id, :official_station_name, :slug, :station_number, :fy17_parent_station_complexity_level)
+    @clinical_resource_hubs = ClinicalResourceHub.cached_clinical_resource_hubs
+    visn_id = Visn.where(number: params["number"].to_i).first.id
```

Comment:
I think we can use `.find_by` here. It makes it a little cleaner because you won't have to call `.first` at the end as it only returns one record.

Reply:
Yep that works.